M2/LICENSING-NOTES.md

```diff
@@ -0,0 +1,122 @@
+| -lfactory          | [Singular-Factory](https://www.singular.uni-kl.de/)                          | GPL-2.0-or-later                      |
+| -lffi              | [libffi](https://sourceware.org/libffi/)                                     | MIT                                   |
+| -lflint            | [FLINT](https://flintlib.org/)                                               | LGPL-3.0-or-later                     |
+| -lfrobby           | [frobby](https://github.com/Macaulay2/frobby)                                | GPL-2.0-or-later                      |
+| -lgc               | [Boehm gc](https://www.hboehm.info/gc/)                                      | Boehm-GC                              |
+| -lgdbm             | [GDBM](https://www.gnu.org/software/gdbm/)                                   | GPL-3.0-or-later                      |
+| -lgivaro           | [Givaro](https://casys.gricad-pages.univ-grenoble-alpes.fr/givaro/)          | CECILL-B                              |
+| -lgmp              | [GMP](https://gmplib.org/)                                                   | LGPL-3.0-or-later OR GPL-2.0-or-later |
+| -llapack           | [LAPACK](https://www.netlib.org/lapack/)                                     | implementation specific               |
+| -lmathic           | [mathic](https://github.com/Macaulay2/mathic)                                | LGPL-2.0-or-later                     |
+| -lmathicgb         | [mathicgb](https://github.com/Macaulay2/mathicgb)                            | GPL-2.0-or-later                      |
+| -lmemtailor        | [memtailor](https://github.com/Macaulay2/memtailor)                          | BSD-3-Clause                          |
+| -lmpfi             | [MPFI](https://gitlab.inria.fr/mpfi/mpfi)                                    | LGPL-2.1-or-later                     |
+| -lmpfr             | [MPFR](https://www.mpfr.org/)                                                | LGPL-3.0-or-later                     |
+| -lmps              | [MPSolve](https://numpi.dm.unipi.it/scientific-computing-libraries/mpsolve/) | GPL-3.0-or-later                      |
+| -lnormaliz         | [Normaliz](https://www.normaliz.uni-osnabrueck.de/)                          | GPL-3.0-or-later                      |
+| -lntl              | [NTL](https://libntl.org/)                                                   | LGPL-2.1-or-later                     |
+| -lpython3          | [Python](https://www.python.org/)                                            | PSF-2.0                               |
+| -lreadline         | [Readline](https://www.gnu.org/software/readline)                            | GPL-3.0-or-later                      |
+| -ltbb              | [oneTBB](https://uxlfoundation.github.io/oneTBB/)                            | Apache-2.0                            |
+
+Programs we include with our distribution:
```

Comment:
Do we?

Reply:
Potentially.  I suppose this should be something like "... we may include ..."

Reply:
Which distribution includes these? A build target that bundles these programs is not the same as a distribution.

Reply:
The RHEL rpms include most (maybe all) of them, since so few are available in the official repos.